atomic_reactor/constants.py

```diff
@@ -148,6 +148,9 @@
 KOJI_MAX_RETRIES = 120
 KOJI_RETRY_INTERVAL = 60
 KOJI_OFFLINE_RETRY_INTERVAL = 120
+# max retries for locking remote host slots
+REMOTE_HOST_MAX_RETRIES = 5
+REMOTE_HOST_RETRY_INTERVAL = 30
```

Comment:
this is too long, try every 5 sec

also try at least 10 times

Reply:
updated to 10 and 5 respectively